lib/lte_link_control/modules/ncellmeas.c

```diff
@@ -610,6 +610,17 @@ static int parse_ncellmeas(const char *at_response, struct lte_lc_cells_info *ce
 	return err;
 }
 
+static void dispatch_empty_neighbor_cell_meas_event(void)
```

Comment:
Nit: I would prefer if the signature aligned with the other functions, following a similar namespacing format. Usually the verb is at the end of the name, example `ncellmeas_empty_event_dispatch`. Having a consistent style makes the code a bit easier to read/parse.